elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -471,16 +480,17 @@ private String getInheritanceKey(String subClass, String superClass) {
     }
 
     private void registerPreSecurityObservers() {
+
         this.distinctLifecycleEvents
                 .filter(CRUDEvent::isReadEvent)
-                .subscribe(new LifecycleHookInvoker(dictionary, OnReadPreSecurity.class));
+                .subscribeWith(new LifecycleHookInvoker(dictionary, OnReadPreSecurity.class, true));
 
         this.distinctLifecycleEvents
                 .filter(CRUDEvent::isUpdateEvent)
-                .subscribe(new LifecycleHookInvoker(dictionary, OnUpdatePreSecurity.class));
+                .subscribeWith(new LifecycleHookInvoker(dictionary, OnUpdatePreSecurity.class, true));
 
         this.distinctLifecycleEvents
                 .filter(CRUDEvent::isDeleteEvent)
-                .subscribe(new LifecycleHookInvoker(dictionary, OnDeletePreSecurity.class));
+                .subscribeWith(new LifecycleHookInvoker(dictionary, OnDeletePreSecurity.class, true));
```

Comment:
Why do pre-security hooks not `throwOnError`?

Reply:
JavaRX has different exception handling for a normal subject and a replay subject when the Observer blows up.  The latter swallows the exception and invokes the uncaught exception handler (not what we want) whereas the former propagates the exception to the `onNext` caller of the subject.  

`onError` is only called if the Observable had an exception.  It doesn't seem to address the case where the Observer blows up.

I'm not sure if I'm misapplying reactiveX here - or reactiveX just has strange semantics.  In either case, this is the best I could come up with.  Open to suggestions.